### PR TITLE
Replace airtable iframe with role cards

### PR DESCRIPTION
**Pull request by powell759**

### CODE REVIEW

**Comment by powell759** [comment]
Noticing in the meantime we've created a standard card component.  (https://github.com/HF-RapidResponse/volunteer-portal/pull/96)

Will follow this PR with one to refactor this on that component.

File: client/src/components/Roles.js [hunk 1]
```diff
@@ -1,13 +1,63 @@
...
+      >
+        <h2 className="header-3-section-lead">Oops:</h2>
+        <h2 className="header-3-section-breaker">Error loading initiatives.</h2>
+        <p>Please try again later.</p>
+      </Col>
+    );
+  }
+
+  // Build 'em if you got 'em
+  const roleCards = roles.map((role) => RoleCard(role));
```
Comment (ecaoile) [inline client/src/components/Roles.js]: RoleCard.js is yelling about needing a unique key in the console, but I think the unique key needs to be assigned here.

File: client/src/components/RoleCard.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import '../styles/roles.scss';
+import '../styles/bootstrap-overrides.scss';
+import {Button} from 'react-bootstrap';
+
+function RoleCard(role) {
+  return (
+    <div className="role-card">
+      <div className="card-content">
+        <h3>{role.role_name}</h3>
+        <p class="role-highlight">
```
Comment (ecaoile) [inline client/src/components/RoleCard.js]: needs to be className instead of class

File: client/src/components/Roles.js [hunk 3]
```diff
@@ -16,17 +66,9 @@ function Roles() {
...
-        className="airtable-embed"
-        src="https://airtable.com/embed/shrZmR6ahgCsyUBGy?backgroundColor=greenLight&amp;viewControls=on"
-        frameBorder="0"
-        width="100%"
-        height="1000"
-        onLoad={() => setLoading(false)}
-        title="volunteer-listings"
-        styles="background: transparent; border: 1px solid #ccc;"
-      ></iframe>
+      <div class="role-card-container">
```
Comment (ecaoile) [inline client/src/components/Roles.js]: needs to be className instead of class

File: client/src/components/RoleCard.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+          Priority: {role.priority}
+          <br />
+          Weekly time commitment: {role.min_time_commitment}-{role.max_time_commitment} hrs
+        </p>
+        <p class="role-overview">{role.overview}</p>
```
Comment (ecaoile) [inline client/src/components/RoleCard.js]: needs to be className instead of class

**Review by ecaoile — CHANGES_REQUESTED** [review]
UI looks great. Just needs a few small changes to make React happy

File: client/src/components/RoleCard.js (hunk 4 above)
Comment (powell759) [reply client/src/components/RoleCard.js]: done

File: client/src/components/Roles.js (hunk 3 above)
Comment (powell759) [reply client/src/components/Roles.js]: done

File: client/src/components/RoleCard.js (hunk 2 above)
Comment (powell759) [reply client/src/components/RoleCard.js]: done

File: client/src/components/Roles.js (hunk 1 above)
Comment (powell759) [reply client/src/components/Roles.js]: Can you paste the error here? I'm not seeing it and not sure what you/react mean by needing a unique key.

Comment (ecaoile) [reply client/src/components/Roles.js]: works now after latest commit

**Review by ecaoile — APPROVED** [review]

**Pull request merged**